### PR TITLE
fix(UVE): Copy URL Button doesnt work

### LINKED ISSUE #29570: UVE: Copy URL Button doesnt work
### Parent Issue

_No response_

### Task

https://github.com/user-attachments/assets/761990a2-1cc7-4478-9173-506e606bdf35

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### Acceptance Criteria

_No response_

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by rjvelazco**
### Proposed Changes
* Make the Copy URL Button work

### Video

https://github.com/user-attachments/assets/3655ec64-2b7d-4616-ad8f-53fcd8aeef93

This PR fixes: #29570

### CODE REVIEW

**Review by valentinogiardino — APPROVED** [review]

File: core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts [hunk 1]
```diff
@@ -271,13 +271,13 @@ export function createFavoritePagesURL(params: {
...
 /**
- * Create a pure URL from the params
  *
+ * @description Create a full URL with the clientHost
  * @export
  * @param {DotPageApiParams} params
  * @return {*}  {string}
  */
-export function createPureURL(params: DotPageApiParams): string {
+export function createFullURL(params: DotPageApiParams): string {
```
Comment (zJaaal) [inline core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts:280]: We need to also send the `host_id` to support multisite

**Review by zJaaal — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts (hunk 1 above)
Comment (rjvelazco) [reply core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts:280]: Out of the issue's scope. It may need to be addressed in a new ticket.

Comment (rjvelazco) [reply core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts:280]: After a talk with @zJaaal, we are going to address the `host_id` here

Comment (rjvelazco) [reply core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts]: Done!

**Review by zJaaal — APPROVED** [review]

**Pull request merged**